extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -48,19 +51,31 @@
   private static final Logger LOG = new Logger(OpenCensusProtobufInputRowParser.class);
 
   private static final String SEPARATOR = "-";
-  public static final String NAME = "name";
-  public static final String VALUE = "value";
-  public static final String TIMESTAMP_COLUMN = "timestamp";
+  private static final String DEFAULT_METRIC_DIMENSION = "name";
+  private static final String VALUE = "value";
+  private static final String TIMESTAMP_COLUMN = "timestamp";
+  private static final String DEFAULT_RESOURCE_PREFIX = "resource.";
   private final ParseSpec parseSpec;
   private final List<String> dimensions;
 
+  private final String metricDimension;
+  private final String metricLabelPrefix;
+  private final String resourceLabelPrefix;
```

Comment:
I just realized you didn't add any @UsonProperty annotated getters for those properties, which means they will be left out when serializing to json. This is required for nodes to communicate the parser spec to each-other

Reply:
@xvrl Created PR: https://github.com/confluentinc/druid/pull/15
Added additional test cases and `labels` for `distribution` metric too.